infra/dockerfile.http_worker

```diff
@@ -2,11 +2,9 @@ ARG BUGBUG_VERSION=latest
 
 FROM mozilla/bugbug-base:$BUGBUG_VERSION
 
-COPY requirements.txt /code/http_service/
-
-RUN pip install --disable-pip-version-check --quiet --no-cache-dir -r /code/http_service/requirements.txt
-
-COPY . /code/http_service/
+# Setup http service as package
+COPY http_service /code/http_service
+RUN pip install --disable-pip-version-check --quiet --no-cache-dir /code/http_service
```

Comment:
This way we will reinstall dependencies even when we don't need to (e.g. if we change a python file).

Reply:
We still need to trigger the http_service's setup.py. What would you prefer ? pip install requirements & then a pip install for http_service module ?

Reply:
Just a side note: that build is slow because of the models loading/checking, not really because of the deps setup

Reply:
> We still need to trigger the http_service's setup.py. What would you prefer ? pip install requirements & then a pip install for http_service module ?

Yeah, similarly to what we do for the bugbug-base

Reply:
Done